dev/com.ibm.ws.concurrent.persistent_fat_multiple/publish/servers/com.ibm.ws.concurrent.persistent.fat.multiple/server.xml

```diff
@@ -0,0 +1,49 @@
+<!--
+    Copyright (c) 2019 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server>
+  <featureManager>
+    <feature>jndi-1.0</feature>
+    <!-- <feature>osgiConsole-1.0</feature> -->
+    <feature>persistentExecutor-1.0</feature>
+    <feature>servlet-3.1</feature>
+    <feature>componenttest-1.0</feature>
+  </featureManager>
+
+  <include location="../fatTestPorts.xml"/>
+
+  <persistentExecutor id="executor1" jndiName="concurrent/executor1" pollInterval="4s" retryLimit="1" pollSize="100"/>
+  <persistentExecutor id="executor2" jndiName="concurrent/executor2" taskStoreRef="defaultDatabaseStore" pollInterval="4s" retryLimit="1"/>
+
+  <databaseStore id="defaultDatabaseStore" keyGenerationStrategy="SEQUENCE"/>
+  
+  <library id="AnonymousJDBCLib">
+    <fileset dir="${shared.resource.dir}/jdbc" includes="${env.DB_DRIVER}"/>
+  </library>
+  
+  <!-- Default database, properties changed by FATRunner during database rotation. -->
+  <dataSource id="DefaultDataSource">
+    <jdbcDriver libraryRef="AnonymousJDBCLib"/>
+  	<properties.derby.embedded createDatabase="create" databaseName="memory:persistmultidb" user="userx" password="passx"/>
```

Comment:
to go along with some of the earlier comments, lets use a generic properties element with variables like this:
```xml
<properties hostname="${DB_HOST}" portNumber="${DB_PORT}" databaseName="${DB_NAME}"
                     user="${DB_USER}" password="${DB_PASSWORD}" createDatabase="create"/>
```
This might give some warnings in logs because `portNumber` is not a valid attribute for Derby Embedded, and `create` is not a valid attribute for all DBs besides Derby Embedded, but we can ignore these on `stopServer()` and it will simplify the code because we won't need to deal with the server config API anymore

Reply:
This actually causes Exceptions to be thrown when attempting to create a connection.
`java.lang.AssertionError: Expected response 200, received 500 (Internal Server Error) while connecting to http://localhost:8010/persistmultitest?testMethod=testConnectionSharing.`
A 500 is returned since the warning about incorrect properties is thrown as a SQL Exception by the application 
`java.sql.SQLException: DSRA8020E: Warning: The property &#39;createDatabase&#39; does not exist on the DataSource class com.ibm.db2.jcc.DB2XADataSource.`

Reply:
oh, that's because we have `<variable name="onError" value="FAIL"/>` in the config. If you remove this line of config, then we ought to just get the warning which we can ignore on `stopServer()`